src/videotile/DefaultVideoTile.ts

```diff
@@ -181,7 +181,16 @@ export default class DefaultVideoTile implements DevicePixelRatioObserver, Video
 
   bindVideoElement(videoElement: HTMLVideoElement | null): void {
     let tileUpdated = false;
+    // If videoElement is null, we are doing an unbinding
     if (this.tileState.boundVideoElement !== videoElement) {
+      if (videoElement === null) {
+        DefaultVideoTile.disconnectVideoStreamFromVideoElement(
+          this.tileState.boundVideoElement,
+          false
+        );
+        this.tileState.videoElementCSSWidthPixels = null;
```

Comment:
Could we keep the logic below as before? Not sure why we want to move it here. We will no longer handle the case where boundVideoElement is null and videoElement is null but the height and width are not.

Reply:
youre right, I moved it back and just added this condition